site/realm/data_sources/mongodb-atlas/aiidprod/incidents/rules.json

```diff
@@ -32,25 +32,48 @@
             "search": true,
             "additional_fields": {}
         },
+        {
+            "name": "Add Variant",
```

Comment:
Is this a new role? Should this be part of the default role instead?

Reply:
I prefer to add a separate role to be more precise on the purpose of these new rules. If we want to remove the anonymous Variant creation in the future we can just remove this role and keep the `Default` role unmodified.